src/modules/webhooks/get-interaction/1.0/module.json

```diff
@@ -1,40 +1,1053 @@
+      "value": "$.interaction.details.languageCodes",
+      "defaultValue": "detailsLanguageCodes"
+    },
+    {
+      "name": "Source Details Name",
+      "value": "$.interaction.sourceDetails.name",
+      "defaultValue": "sourceDetailsName"
+    },
+    {
+      "name": "Source Details Email",
+      "value": "$.interaction.sourceDetails.email",
+      "defaultValue": "sourceDetailsEmail"
+    },
+    {
+      "name": "Source Details Person Sms Device Id",
+      "value": "$.interaction.sourceDetails.person.sms.deviceId",
+      "defaultValue": "sourceDetailsPersonSmsDeviceId"
+    },
+    {
+      "name": "Source Details Person Sms Person Id",
+      "value": "$.interaction.sourceDetails.person.sms.personId",
+      "defaultValue": "sourceDetailsPersonSmsPersonId"
+    },
+    {
+      "name": "Source Details Person Sms Session Id",
+      "value": "$.interaction.sourceDetails.person.sms.sessionId",
+      "defaultValue": "sourceDetailsPersonSmsSessionId"
+    },
+    {
+      "name": "Source Details Person Sms Phone Number",
+      "value": "$.interaction.sourceDetails.person.sms.phoneNumber",
+      "defaultValue": "sourceDetailsPersonSmsPhoneNumber"
+    },
+    {
+      "name": "Source Details Person Phone Device Id",
+      "value": "$.interaction.sourceDetails.person.phone.deviceId",
+      "defaultValue": "sourceDetailsPersonPhoneDeviceId"
+    },
+    {
+      "name": "Source Details Person Phone Person Id",
+      "value": "$.interaction.sourceDetails.person.phone.personId",
+      "defaultValue": "sourceDetailsPersonPhonePersonId"
+    },
+    {
+      "name": "Source Details Person Phone Session Id",
+      "value": "$.interaction.sourceDetails.person.phone.sessionId",
+      "defaultValue": "sourceDetailsPersonPhoneSessionId"
+    },
+    {
+      "name": "Source Details Person Phone Phone Number",
+      "value": "$.interaction.sourceDetails.person.phone.phoneNumber",
+      "defaultValue": "sourceDetailsPersonPhonePhoneNumber"
+    },
+    {
+      "name": "Source Details Person Device Id",
+      "value": "$.interaction.sourceDetails.person.deviceId",
+      "defaultValue": "sourceDetailsPersonDeviceId"
+    },
+    {
+      "name": "Source Details Person Person Id",
+      "value": "$.interaction.sourceDetails.person.personId",
+      "defaultValue": "sourceDetailsPersonPersonId"
+    },
+    {
+      "name": "Source Details Person Session Id",
+      "value": "$.interaction.sourceDetails.person.sessionId",
+      "defaultValue": "sourceDetailsPersonSessionId"
+    },
+    {
+      "name": "Source Details Person Phone Number",
+      "value": "$.interaction.sourceDetails.person.phoneNumber",
+      "defaultValue": "sourceDetailsPersonPhoneNumber"
+    },
+    {
+      "name": "Source Details Phone Number",
+      "value": "$.interaction.sourceDetails.phoneNumber",
+      "defaultValue": "sourceDetailsPhoneNumber"
+    },
+    {
+      "name": "Source Details Phone Number Formatted Type",
```

Comment:
Lets ditch the `Formatted` part from the name and defaultValue for all these phone number ones